### DISCUSSION TITLE
In which field should I fill the reference product?

### DISCUSSION THREAD

**Discussion by stephane-klein** [medusajs/medusa#3530] [Q&A] [answered]
Hello,

I am trying to import a series of products into *Medusa* via *Medusa admin* and I have the following question: where fill the product reference?

![Capture d’écran du 2023-03-20 17-58-44](https://user-images.githubusercontent.com/54498080/226414250-9730ae92-6056-4022-9fb0-9343a01f6795.png)

I don't plan to enter the product reference value in "handle" field because I consider this field to be used for a [slug](https://en.wikipedia.org/wiki/Clean_URL#Slug).

For example, the product reference of [Lenovo ThinkPad T14s Gen 3](https://www.amazon.com/Lenovo-ThinkPad-T14s-21CQ000LUS-Notebook/dp/B0BHWRS2QK/) is `21CQ000LUS`. Where can I fill `21CQ000LUS` value?

Best regards,  
Stéphane

**Comment by srindom** [comment, marked as answer]
Hi @stephane-klein, looks like you are looking for the SKU field. In Medusa, there are Products and Product Variants - SKUs live on Product Variants so you have to add an option and an option value and you will then be able to specify the parameters for the variant. 

Let me know if that works :)

**Reply by stephane-klein** [reply]
> Let me know if that works :)

Yes, it works.
